vnext/local-cli/runWindows/utils/VSProjectUtils.ps1

```diff
@@ -0,0 +1,153 @@
+
+    $packageXml.Save($PackagesConfigPath)
+
+    Write-Host " done."
+}
+
+# Loads the packages.config file at $PackagesConfigPath
+# If $CheckConfig, check the file exists first
+function Load-PackagesConfigXml {
+    param(
+        [string] $PackagesConfigPath,
+        [bool] $CheckConfig = $True
+    )
+
+    if ($CheckConfig) {
+        Check-PackagesConfigXml -PackagesConfigPath $PackagesConfigPath
+    }
+
+    Write-Host
+    Write-Host "Loading " -NoNewLine
+    Write-Host -ForegroundColor White $PackagesConfigPath -NoNewLine
+    Write-Host "..." -NoNewLine
+
+    [xml]$content = Get-Content -Path $PackagesConfigPath
+
+    Write-Host " done."
+
+    $content.packages.package | foreach {
+            Write-Host "Found package " -NoNewLine
+            Write-Host -ForegroundColor White $_.id -NoNewLine
+            Write-Host " version " -NoNewLine
+            Write-Host -ForegroundColor White $_.version -NoNewLine
+            Write-Host "."
+        }
+
+    return $content;
+}
+
+# Update package $PackageName to version $PackageVersion in the packages.config
+# file at $PackagesConfigPath
+# If $CheckConfig, check the file exists first
+function Update-PackagesConfigXml {
+    param(
+        [string] $PackagesConfigPath,
+        [string] $PackageName,
+        [string] $PackageVersion,
+        [bool] $CheckConfig = $True
+    )
+
+    if ($CheckConfig) {
+        Check-PackagesConfigXml -PackagesConfigPath $PackagesConfigPath
+    }
+
+    Write-Host
+    Write-Host "Updating package " -NoNewLine
+    Write-Host -ForegroundColor White $PackageName -NoNewLine
+    Write-Host " to version " -NoNewLine
+    Write-Host -ForegroundColor White $PackageVersion -NoNewLine
+    Write-Host "..."
+
+    & nuget.exe update "$PackagesConfigPath" -Id "$PackageName" -Version "$PackageVersion"
+}
+
+# Add package $PackageName version $PackageVersion to the packages.config
+# file at $PackagesConfigPath
+# If $CheckConfig, check the file exists first
+function Add-PackagesConfigXml {
+    param(
+        [string] $PackagesConfigPath,
+        [string] $PackageName,
+        [string] $PackageVersion,
+        [bool] $CheckConfig = $True
+    )
+
+    $packageXml = Load-PackagesConfigXml -PackagesConfigPath $PackagesConfigPath
+
+    $existingPackageNode = $packageXml.selectSingleNode("packages/package[@id=""$PackageName""]")
+
+    if ($existingPackageNode) {
+        Write-Host
+        Write-Host "Nothing to add, package " -NoNewLine
+        Write-Host -ForegroundColor White $PackageName -NoNewLine
+        Write-Host " is already present."
+    } else {
+        Write-Host
+        Write-Host "Adding package " -NoNewLine
+        Write-Host -ForegroundColor White $PackageName -NoNewLine
+        Write-Host "..." -NoNewLine
+
+        $packagesNode = $packageXml.selectSingleNode('packages')
+        $newPackageNode = $packagesNode.appendChild($packageXml.CreateElement('package'))
+        $newPackageNode.SetAttribute('id', $PackageName)
```

Comment:
we should also add targetFramework="native" for C++ projects

Reply:
but that only works for c++. c# needs something different. by not specifying it, nuget does the right thing for the target project, without us having to maintain that logic